website/docs/d/custom_dns_configuration_cluster_aws.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: custom_dns_configuration_cluster_aws"
+sidebar_current: "docs-mongodbatlas-datasource-custom_dns_configuration_cluster_aws"
+description: |-
+    Describes a Custom DNS Configuration for Atlas Clusters on AWS.
+---
+
+# mongodbatlas_custom_dns_configuration_cluster_aws
+
+`mongodbatlas_custom_dns_configuration_cluster_aws` describes a Custom DNS Configuration for Atlas Clusters on AWS.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find **group_id** in the official documentation.
+
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_custom_dns_configuration_cluster_aws" "test" {
+	project_id                  = "<project-id>"
+	enabled                     = true
+}
+
+data "mongodbatlas_custom_dns_configuration_cluster_aws" "test" {
+    project_id = mongodbatlas_custom_dns_configuration_cluster_aws.test.id
```

Comment:
I don't think this is right?    would it not be project_id = mongodbatlas_custom_dns_configuration_cluster_aws.test.project_id ?

Reply:
Both case are correct, I mean, project_id and id has the same data.

Reply:
Ah, that's right - project_id doesn't exist.  This is not one of the resources with an Atlas id and Terraform id (which really confuses users and obviously me as well).